sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -5,6 +5,10 @@ extensions:
     - name: "openshift-ansible"
     - name: "aos-cd-jobs"
   actions:
+    - type: "forward_parameters"
+      parameters:
+        - ORIGIN_TARGET_BRANCH
```

Comment:
Hrm, this doesn't solve https://github.com/openshift/aos-cd-jobs/issues/315 holistically for all jobs or even all repos for this job :(